CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 2.18.0-rc.0 / 2020-04-24
+
+* [CHANGED] Use .UTC timezeone everywhere we use time.Unix. #7066
+* [CHANGED] Federation: It does not use remote storages anymore as designed initially. #7077
+* [CHANGED] Rules: `rule_evaluations_total` and `rule_evaluation_failures_total` have `rule_group` label now. #7094
+* [CHANGED] Moved to Go 1.14.2. #7100
+* [FEATURE] Tracing(!): Added experimental Jaeger support #7148
```

Comment:
This should not be documented as it will not stay for long and is not documented, also it is not relevant for users

Reply:
I think actually it  might be the most relevant part of this release (: Except maybe Federation (potentially breaking) change

Reply:
It is something mainly aimed at developers, not users, and that we explicitely did not document because we do not want to support it in short or long term. Adding it here would imply that we need documentation otherwise we will get many questions.

WDYT?

Also, cc @brian-brazil on this one.

Reply:
I see your point, we can mention that documentation will be added at some point once production-ready.

There is no point in adding experimental feature without any mention (: Who will experiment then?

cc @RichiH @juliusv @brancz

Reply:
I have said what I had to say, let's get more input. But it is not experimental, it is stop-gap until we have opentelemetry.

Reply:
It should be listed, but given the odd situation I'd downplay it and put it as an enhancement.

Experimental does not mean not production ready, it means we're allowed break it - or in this case completely remove it in future.

Reply:
I think the feature of `Tracing` is here to stay, and should be in the release notes. The fact that it is Jaeger might be short term, but the experimental marker means we can change the implementation at any time. What is the `(!)` meant to convey?

Reply:
Yeah, (!) is not something the changelog should contain.

Reply:
The `(!)` is still here.

Reply:
I'd also remove the `(!)`.

> Experimental does not mean not production ready, it means we're allowed break it - or in this case completely remove it in future.

Just FTR: I don't know where @brian-brazil is taking this from. I have certainly marked many features experimental not _only_ to mark it as something that might change later without a major version bump but _also_ to not set any expectation it's production ready. We don't need to discuss the details here. I have added an item to the dev summit agenda already. I mostly want to file my objection here to avoid @brian-brazil to once more use an imaginary past consensus in future arguments.

Reply:
I'm working off https://prometheus.io/docs/prometheus/latest/stability/
Our language here has historically not been the colloquial usage, so I want to avoid confusion.

Reply:
That document doesn't state anything about production readiness.

> Our language here has historically not been the colloquial usage, so I want to avoid confusion.

Once more, you are using "we/us/our" when you mean "I/me/my". Or if you really want to speak for all of us, you should at least mark that you are describing your personal impression of what we are doing, which historically has deviated repeatedly from the impression of the rest of us.

Reply:
> That document doesn't state anything about production readiness.

Exactly, and I don't believe we have any docs that do.

> Once more, you are using "we/us/our" when you mean "I/me/my".

I mean "we" here. We as a group have been sloppy in our usage of the word "experimental", with the prime example being remote write which has been both experimental and production ready for some years. This was never an explicit consensus or decision that I know of, rather a thing that naturally happened over time. It'd be good to discuss how to clean this up at the next dev summit, as this situation has caused some user confusion over the years.

Anyway, in this particular case the words in the changelog entry seem fine.

Reply:
> > That document doesn't state anything about production readiness.
>
> Exactly, and I don't believe we have any docs that do.

I am not a native speaker, but I'm pretty sure that the word "experimental" generally does not mean "as production ready as a stable feature". Should our intention be that the only implication of "experimental" is that those feature do not fall under semantic versioning, then we should better call that out. As we don't call that out, and the document you quoted doesn't even look like a definition of "experimental", much less an exhaustive one, I don't get your line of argument, which with you have now derailed not only this PR but also the node_exporter TLS discussion.

> I mean "we" here.

It's hopeless… but I have realized that early 2017 already. What am I even doing here?
I'll stop here and now.

Reply:
Please, let's not stop @beorn7 @brian-brazil ! 

We might have different opinions, but by **doing nothing** and just hiding we won't find a viable middle ground where everyone is happy! And I am pretty sure we can all find that spot :hugs: 

@beorn7, please give you feedback as often as possible! Your opinion is extremely valuable for the project, community (and definitely for me!), and the same is for @brian-brazil 's opinion. We all can learn only by communication ([as Go, humans do not share memory](https://blog.golang.org/codelab-share))!

I think it's totally fair to take discussion about `experimental` meaning to DevSummit. I will make sure the topic is there. No matter what was said before or now, it's not clear for some of us, so let's agree on this at next meeting.

My 2c: @brian-brazil you are amazing, but I will be honest here: It's a bit rude to say `we/us` for the topic where clearly at least two people (me and @beorn7) among those `we` are not agreeing with the opinion. (: PLUS people ask you many times to not say `we`...

**TBH, no matter if by saying `we` @brian-brazil you mean 90% or 1%. population If your friends kindly ask you to not use one word that in some way hurts them (even though you don't mean to), let's maybe not use that. It is really the same as the word `he` or `black` for gender/race contexts.** Can we kindly ask you?  :hugs:

And let's discuss further on our meetings! :pray: